floodlight/models/utils.py

```diff
@@ -0,0 +1,38 @@
+import numpy as np
+
+from floodlight import XY
+
+
+def _exclude_x_ids(xy: XY, xIDs_to_exclude: list) -> np.ndarray:
+    """Returns an array of boolean values indicating which xIDs to exclude.
```

Comment:
that's not correct, it returns a list of indices to include, not to exclude and of columns, not of xIDs. thus, a more appropriate function name and description would be nice